kernel/src/main/java/org/kframework/parser/inner/kernel/Grammar.java

```diff
@@ -58,6 +58,14 @@
     /** The set of "root" NonTerminals */
     private BiMap<String, NonTerminal> startNonTerminals = HashBiMap.create();
 
+    public NonTerminal nullNT() {
+      return get("_Null");
+    }
+
+    public Grammar() {
+      add(new NonTerminal("_Null"));
+    }
```

Comment:
We're sure these "special" nonterminals cannot show up anywhere in a user's grammar?

Reply:
I'm sure. Sort names can't begin with an underscore.